Formula/kubernetes-cli@1.22.rb

```diff
@@ -0,0 +1,63 @@
+class KubernetesCliAT122 < Formula
+  desc "Kubernetes command-line interface"
+  homepage "https://kubernetes.io/"
+  url "https://github.com/kubernetes/kubernetes.git",
+      tag:      "v1.22.4",
+      revision: "b695d79d4f967c403a96986f1750a35eb75e75f1"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    regex(/^v?(1\.22(?:\.\d+)+)$/i)
+  end
+
+  keg_only :versioned_formula
+
+  # https://kubernetes.io/releases/patch-releases/#1-22
+  deprecate! date: "2022-10-28", because: :deprecated_upstream
```

Comment:
Let's do it like this so we have the full information ready to go
```suggestion
  deprecate! date: "2022-08-28", because: :deprecated_upstream
#  disable! date: "2022-10-28", because: :deprecated_upstream
```